src/cljs/ataru/virkailija/editor/component.cljs

```diff
@@ -596,6 +619,54 @@
                                    (dispatch [:editor/select-koodisto-options uri version title path]))}
                       title]]))]])])))
 
+(defn- custom-answer-options [path question-group-element? editable?]
+  (let [languages  @(subscribe [:editor/languages])
+        value      @(subscribe [:editor/get-component-value path])
+        field-type (:fieldType value)]
+    (seq [
+          ^{:key "options-input"}
+          [:div.editor-form__multi-options-container
+           (map-indexed (fn [idx _]
+                          (dropdown-option idx [path :options idx] path languages
+                                           :editable? editable?
+                                           :include-followup? (and (not question-group-element?)
+                                                                   (some #{field-type} ["dropdown" "multipleChoice" "singleChoice"]))))
+             (:options value))]
+          (if editable?
+            ^{:key "options-input-add"}
+            [:div.editor-form__add-dropdown-item
+             [:a
+              {:on-click (fn [evt]
+                           (.preventDefault evt)
+                           (dispatch [:editor/add-dropdown-option path]))}
+              [:i.zmdi.zmdi-plus-square] " Lisää"]])])))
+
+(defn koodisto-answer-options [id path selected-koodisto question-group-element?]
+  (let [opened? (r/atom false)]
+    (fn [id path selected-koodisto question-group-element?]
+      (let [languages             @(subscribe [:editor/languages])
+            value                 @(subscribe [:editor/get-component-value path])
+            field-type            (:fieldType value)
+            editable?             false
+            hide-koodisto-options (fn [evt]
+                                    (reset! opened? false))
+            show-koodisto-options (fn [evt]
+                                    (swap! opened? (fn [old]
+                                                     (if (not old)
+                                                       (dispatch [:editor/fetch-koodisto-for-component-with-id id selected-koodisto]))
+                                                     true)))]
+        (if (not @opened?)
+          [:div.editor-form__show-koodisto-values
+           [:a
+            {:on-click show-koodisto-options}
+            [:i.zmdi.zmdi-chevron-down] " Näytä vastausvaihtoehdot"]]
+          [:div
+           [:div.editor-form__show-koodisto-values
+            [:a
+             {:on-click hide-koodisto-options}
+             [:i.zmdi.zmdi-chevron-up] " Sulje vastausvaihtoehdot"]]
+           (custom-answer-options path question-group-element? editable?)])))))
```

Comment:
Pitäis olla `[custom-answer-options ...` koska siellä on subscriptioneita. Vaihtoehtosesti vois välittää `language`:n parametrina, jolloin noi subscriptionit vois poistaa tuolta `custom-answer-options`:sta.